airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -458,6 +467,82 @@ def run_check_dbt_command(normalization_image: str, command: str, cwd: str, forc
             command = f"{command} --full-refresh"
         print("Executing: ", " ".join(commands))
         print(f"Equivalent to: dbt {command} --profiles-dir={cwd} --project-dir={cwd}")
+
+        error_count = 0
+        with open(os.path.join(cwd, "dbt_output.log"), "ab") as f:
```

Comment:
is this identical to what `run_dbt_run_operation` is doing? This is a pretty complex bit of logic, so I'd prefer to pull it into a separate function instead of duplicating it

Reply:
Done.